http/headers/Server-Timing.json

```diff
@@ -39,6 +39,41 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "trailer": {
+          "__compat": {
+            "description": "`Server-Timing` as HTTP trailer",
+            "support": {
+              "chrome": {
+                "version_added": false
```

Comment:
Oh, we should probably add the Chromium impl_url here:

https://issues.chromium.org/issues/40811358

Reply:
Good idea, thanks for the remninder

Reply:
added in 7314336174563b2d205c66b0ecc84f6ce4dc80be